example/rnn/cudnn_lstm_bucketing.py

```diff
@@ -76,12 +78,15 @@ def train(args):
     if args.stack_rnn:
         stack = mx.rnn.SequentialRNNCell()
         for layer in range(args.num_layers):
+            dropout = 0.0
+            if layer < (args.num_layers - 1):
+                dropout = args.dropout
             stack.add(mx.rnn.FusedRNNCell(args.num_hidden, num_layers=1,
-                    mode='lstm', prefix='lstm_%d'%layer,
+                    mode='lstm', prefix='lstm_%d'%layer, dropout=dropout,
```

Comment:
@mkolod Do you know if cudnn rnn drops the input or output or both? If it only drops output from non-final layer then this doesn't do anything.